### PR TITLE
Lichess test

### PR DESCRIPTION
**Pull request by TheYoBots**
- Test with all python versions for windows.
- Add token
- MarkZH’s suggestion

Other Notes:
- For some reason it seems to face a bad request error and doesn’t play a move.
- Token only contains required scopes, not all (bot:play, challenge:read, challenge:write). This is the account https://lichess.org/@/Bot_pytests .

### CODE REVIEW

**Comment by AttackingOrDefending** [comment]
There is no reason to change abort to make_move. Moves are made from start in line 46. I just copied the code from lichess-bot.py. The code that you changed aborts every game that can be aborted.

File: tests.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def run_bot(logging_level):
     game_ids = list(map(lambda game: game['gameId'], games))
     for game in game_ids:
         try:
-            li.abort(game)
+            li.make_move(game)
```
Comment (AttackingOrDefending) [inline tests.py]: Change this back to abort.

**Comment by AttackingOrDefending** [comment]
I will make a new branch called test-lichess-test and I will make the required changes and make a new PR here.

**Comment by AttackingOrDefending** [comment]
See https://github.com/AttackingOrDefending/lichess-bot/pull/2. I changed it to every branch to test this. You can add this to this PR also if you want to test this.

**Comment by TheYoBots** [comment]
I’ll just add the token then.

**Comment by AttackingOrDefending** [comment]
Now that I removed the first time that it aborted games, it seems to be working well (https://github.com/AttackingOrDefending/lichess-bot/pull/2/checks?check_run_id=2547064172). Can you test this?

**Pull request closed without merging**